pdns/pdnssec.cc

```diff
@@ -1406,12 +1604,25 @@ try
 
   if(cmds[0] == "create-bind-db") {
 #ifdef HAVE_SQLITE3
-    if(cmds.size() != 2) {
-      cerr << "Syntax: pdnssec create-bind-db fname"<<endl;
+    if(cmds.size() < 2 || cmds.size() > 3) {
+      cerr << "Syntax: pdnssec create-bind-db [overwrite] FNAME"<<endl;
       return 0;
     }
     try {
-      SSQLite3 db(cmds[1], true); // create=ok
+      bool overwrite;
+      string dbname;
+      if(cmds.size() == 2) {
+        overwrite = false;
+        dbname = cmds[1];
+      } else {
+        if (cmds[1] != "overwrite"){
+          cerr << "Syntax: pdnssec create-bind-db [overwrite] FNAME"<<endl;
+          return 0;
+        }
+        overwrite = true;
+        dbname = cmds[2];
+      }
+      SSQLite3 db(dbname, true, overwrite); // create=ok
```

Comment:
Because we now warn that a database already exists, we should also be able to say 'just delete it'